pandas/tools/util.py

```diff
@@ -48,3 +48,60 @@ def compose(*funcs):
     """Compose 2 or more callables"""
     assert len(funcs) > 1, 'At least 2 callables must be passed to compose'
     return reduce(_compose2, funcs)
+
+
+def to_numeric(arg, errors='raise'):
+    """
+    Convert argument to a numeric type.
+
+    Parameters
+    ----------
+    arg : string, array of strings, mixed object array
+    errors : {'ignore', 'raise', 'coerce'}, default 'raise'
+        - If 'raise', then invalid parsing will raise an exception
+        - If 'coerce', then invalid parsing will be set as NaN
+        - If 'ignore', then invalid parsing will return the input
+    box : boolean, default True
+        - If True returns a Series
+        - If False returns ndarray of values.
+
+    Returns
+    -------
+    ret : numeric if parsing succeeded.
+        Return type depends on box
+
+    Examples
+    --------
+    Take separate series and convert to numeric, coercing when told to
+
+    >>> import pandas as pd
+    >>> s = pd.Series(['1.0', '2', -3])
+    >>> pd.to_numeric(s)
+    >>> s = pd.Series(['apple', '1.0', '2', -3])
+    >>> pd.to_numeric(s, errors='ignore')
+    >>> pd.to_numeric(s, errors='coerce')
+    """
+
+    index = None
+    if isinstance(arg, pd.Series):
+        index, name = arg.index, arg.name
+    elif isinstance(arg, (list, tuple)):
+        arg = np.array(arg, dtype='O')
+
+    conv = arg
+    arg = com._ensure_object(arg)
+
+    coerce_numeric = False if errors in ('ignore', 'raise') else True
+
+    try:
+        conv = lib.maybe_convert_numeric(arg,
+                                         set(),
+                                         coerce_numeric=coerce_numeric)
+    except:
+        if errors == 'raise':
+            raise
+
+    if index is not None:
+        return pd.Series(conv, index=index, name=name)
+    else:
+        return conv
```

Comment:
this will return an ndarray if you pass one in, indicate this in the Return Type above (and remove ref to box)